### DISCUSSION TITLE
calendar displays different than widget

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3482] [Q&A]
Hello Pretix team!

I have an event that displays different in widget than when selecting go to shop from the backend of Pretix.
Please check the 2 screenshots below.
Go to shop: https://snipboard.io/3UXnE5.jpg
Widget: https://snipboard.io/HtM0Xb.jpg

How can I display the Select word that exists in Go to shop option?

**Comment by wiffbi** [comment]
This is currently not possible. We added the checkbox-select-button to the normal pretix-shop as a UX improvement, but hesitated to do so in the widget as that probably would have broken lots of customized widgets. Not sure, if we want to add this to the widget e.g. offering an optional setting similar to our custom-plus-minus-rocker-buttons for quantity-inputs.

**Reply by alemao8** [reply]
ok I see.
What if you could create a variable like enable-select so we could include this in the widget if we want to enable it ?
